ecommerce-ui/src/ShoppingCart.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import classNames from 'classnames';
+import ExpansionPanel from 'material-ui/ExpansionPanel';
+import ExpansionPanelDetails from 'material-ui/ExpansionPanel/ExpansionPanelDetails';
+import ExpansionPanelSummary from 'material-ui/ExpansionPanel/ExpansionPanelSummary';
+import ExpansionPanelActions from 'material-ui/ExpansionPanel/ExpansionPanelActions';
+import Typography from 'material-ui/Typography';
+import ShoppingIcon from 'react-icons/lib/ti/shopping-cart';
+import Button from 'material-ui/Button';
+import Divider from 'material-ui/Divider';
+import Badge from 'material-ui/Badge';
+import Styles from 'material-ui/styles';
+
+class ShoppingCartPanel extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleChange(e) {
+    console.log(e.target.value);
+    this.props.onPropertyAdded(e.target.value);
+  }
+
+  printOut(e) {
+    console.log(this.props.property.title);
```

Comment:
Once you get the properties coming into this component as an array, it'll be a lot easier to map over them and print out the shopping cart.